model/gen-ai/events.yaml

```diff
@@ -30,6 +70,79 @@ groups:
     brief: >
       This event describes the assistant message passed to GenAI system or received from it.
     extends: gen_ai.common.event.attributes
+    body:
+      id: gen_ai.assistant.message
+      requirement_level: opt_in
+      type: map
+      fields:
+        - id: content
+          type: undefined
+          stability: experimental
+          brief: >
+            The contents of the tool message.
+          examples: ["The weather in Paris is rainy and overcast, with temperatures around 57°F"]
+          requirement_level: opt_in
+        - id: role
+          type: string
+          stability: experimental
+          brief: >
+            The actual role of the message author as passed in the message.
+          examples: ["assistant", "bot"]
+          requirement_level:
+            conditionally_required: if available and not equal to `assistant`.
+        - id: tool_calls
+          type: map  # TODO: it's an array
```

Comment:
TODO: this is blocker - need to release new weaver that supports it

Reply:
Is this still blocked and the PR ready for review?

Reply:
the PR is ready for review, but the tooling is not ready for us to define array of map type. So even if you review, we can't merge it yet

Reply:
fixed now that tooling supports it